### ISSUE TITLE
Footer Nav Error

### ISSUE THREAD

**Issue by tuck0066** [tuck0066/kraken-good-accessibility#1]
@JessJodoin

Hi Jess!  I am on the Kracken one and am having issues with the navigation for the footer.  Markbot is showing that as my only error but mine doesn't look like the screen shot.  I am on slack as well.

**Comment by JessJodoin** [comment]
Can you take a screenshot of the issue from markbot?

**Comment by tuck0066** [comment]
@JessJodoin

![Screenshot (583)](https://user-images.githubusercontent.com/70998021/95487535-73d95500-0962-11eb-98a9-966f3300871f.png)


I was also hoping you could tell me again how you select all those <P>s so I can fix the spacing.  When I use the line edits that is where it positions it.

**Comment by JessJodoin** [comment]
The nav issues has to do with assigning roles (`role="navigation"`). You should only use this once in your index and should be used for your main navigation, usually the one in your header, so remove it from the footer navigation.

As for the spacing issues, you might have to go through it manually, but if it's doing that when you auto-indent it might be an issue with a missing closing tag

**Comment by JessJodoin** [comment]
It also might be good to go back to the original document and start again. This assignment doesn't need a lot of added elements, it's just changing the ones already there and adding accessibility things

**Comment by tuck0066** [comment]
@JessJodoin

So keep all those <div> tags in and assign something to them?  You had highlighed several different areas to do the same code to.  How do I do that?  Do I have to keep adding the @JessJodoin or does it just go to you when I comment?

**Comment by JessJodoin** [comment]
Nope you just have to tag me in the first one and then I get notified when you post a comment or anyone else that's tagged posts a comment.

and it's command click - so click and hold command and click your cursor on all the lines of code you want to write on, once you've click all the line you want stop holding command and type

**Comment by JessJodoin** [comment]
What I mean is the way the original index is structured is correct. You just go through and change the `<div>` tags to other elements (like header, footer, article, figure, figcaption, etc.) and then add accessibility things like roles and aria-labels

**Comment by tuck0066** [comment]
kk  Off to do that.  The SVG had really confused me.  I went back to watch the videos and couldn't figure out how they were done.  I winged it and apparently markbot likes it.

**Comment by JessJodoin** [comment]
Yeah it can be tricky. One other thing, you have a sh.exe.stackdump 
I'm not sure what it is but it seems sketchy so maybe delete it

**Comment by tuck0066** [comment]
![Screenshot (584)](https://user-images.githubusercontent.com/70998021/95491125-30351a00-0967-11eb-9943-a5bf10671797.png)

Keeps giving me this error now and won't even start.  LOL

**Comment by JessJodoin** [comment]
Hmm. This might be a good time to zoom call. I'll start one
